tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -10,6 +10,12 @@
 from chainerx_tests import op_utils
 
 
+def _skip_if_native_and_lapack_unavailable(device):
```

Comment:
How about "@pytest.mark.skipif"

Reply:
It seems `device` has to be available before the wrapped function is called in order to use `pytest.mark.skipif`.

Reply:
Yes, this is because it's parameterized over devices and we only want to skip for a particular device, so I don't think it's straight forward.